extensions/podman/src/extension.ts

```diff
@@ -451,10 +452,56 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
 
   // Check on initial setup
   checkDisguisedPodmanSocket(provider);
-  // update the status of the provider if the socket is changed, created or deleted
+
+  // Update the status of the provider if the socket is changed, created or deleted
   disguisedPodmanSocketWatcher = setupDisguisedPodmanSocketWatcher(provider, getSocketPath());
   extensionContext.subscriptions.push(disguisedPodmanSocketWatcher);
 
+  // Compatibility mode status bar item
+  // only available for macOS (for now).
+  if (isMac()) {
+    // Get the socketCompatibilityClass for the current OS.
+    const socketCompatibilityMode = getSocketCompatibility();
+
+    // Create a status bar item to show the status of compatibility mode as well as
+    // create a command so when you can disable / enable compatibility mode
+    const statusBarItem = extensionApi.window.createStatusBarItem();
+    statusBarItem.text = 'Docker Socket Compatibility';
```

Comment:
IMHO the text might be too long for the status bar. Tooltip is already explaining

Reply:
Any suggestions? Maybe just "Socket" with a mini-docker logo? Or just "Docker Socket" (since we show the little "plug" logo anyways which looks like compatibility icon).

Reply:
"Docker Compatibility"